WORKSPACE

```diff
@@ -21,11 +21,13 @@ workspace(name = "vaticle_dependencies")
 # Load @vaticle_dependencies #
 ################################
 
-# Load //build/rust
+# Load //builder/rust
 load("//builder/rust:deps.bzl", rust_deps = "deps")
 rust_deps()
-load("@rules_rust//rust:repositories.bzl", "rust_repositories")
-rust_repositories(version = "nightly", iso_date = "2021-07-01", edition="2018")
+
+load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_register_toolchains")
+rules_rust_dependencies()
+rust_register_toolchains(edition = "2021", include_rustc_srcs = True)
```

Comment:
Update to the latest Rust (fixes build issue in `ortools`, and also is just generally good practice)